packages/docs-app/src/getting-started.md

```diff
@@ -92,8 +92,8 @@ These bundles _do not include_ external dependencies; your application will need
     <script src="https://unpkg.com/react-transition-group@^2.2.1/dist/react-transition-group.min.js"></script>
     <script src="https://unpkg.com/popper.js@^1.12.6/dist/umd/popper.js"></script>
     <script src="https://unpkg.com/react-popper@~0.7.4/dist/react-popper.min.js"></script>
```

Comment:
popper.js and react-popper versions need to change too.

https://github.com/palantir/blueprint/blob/develop/packages/core/package.json#L46-L47

Reply:
Happy to!